gateway-service/src/main/java/org/zowe/apiml/gateway/controllers/GatewayExceptionHandler.java

```diff
@@ -136,6 +138,12 @@ public Mono<Void> handleHttpMediaTypeException(ServerWebExchange exchange, Excep
         return setBodyResponse(exchange, SC_UNSUPPORTED_MEDIA_TYPE, "org.zowe.apiml.common.unsupportedMediaType");
     }
 
+    @ExceptionHandler(SSLException.class)
+    public Mono<Void> handleSslException(ServerWebExchange exchange, SSLException ex) {
+        log.debug("SSL exception on {}: {}", exchange.getRequest().getURI(), ex.getMessage());
```

Comment:
since it's a debug message, maybe better to show the stack trace?

Reply:
ok, it probably makes sense for SSL